extra-dev/packages/coq-corn/coq-corn.8.5.dev/opam

```diff
@@ -0,0 +1,24 @@
+opam-version: "2.0"
+maintainer: "b.a.w.spitters@gmail.com"
+homepage: "http://corn.cs.ru.nl/"
+dev-repo: "git+https://github.com/c-corn/corn.git"
+bug-reports: "https://github.com/c-corn/corn/issues"
+doc: "http://corn.cs.ru.nl/"
+authors: "Corn development team"
+license: "GPL 2"
+build: [
+  ["./configure.sh"]
+  [make "-j%{jobs}%"]
+]
+install: [
+  [make "install"]
+]
+depends: [
+  "ocaml"
+  "coq" {>= "8.5" & <= "8.5.2"}
+  "coq-math-classes"
+]
+synopsis: "The CoRN library. A library for constructive analysis"
+url {
```

Comment:
Please add a checksum for the file here (SHA256 preferable).

Reply:
Actually this package refers to a branch. I added a Git url to make it explicit.